Formula/neovim.rb

```diff
@@ -78,8 +78,8 @@ def install
   def caveats; <<-EOS.undent
       The Neovim executable is called 'nvim'. To use your existing Vim
       configuration:
-          ln -s ~/.vimrc ~/.nvimrc
-          ln -s ~/.vim ~/.nvim
+          ln -s ~/.vimrc ~/.config/nvim/init.vim
```

Comment:
Is one meant to only run one of these commands? Otherwise, it's really confusing and I believe the commands should be in the reverse order. The first command requires that `~/.config/nvim` exists and the seconds requires that it does _not_ exist. Reversing them should at least work in the case that `~/.config/nvim` does not exist (but `~/.config` does).

The old caveat had no such problems so I think it should either be documented that one is supposed to run only one of the commands or put them in the right order.